ml-agents/mlagents/trainers/tests/test_bc.py

```diff
@@ -1,227 +0,0 @@
-import unittest.mock as mock
```

Comment:
Can you just make sure that coverage in non-deleted files didn't go down? I can look at the coverage reports with you tomorrow.

Reply:
Sounds good - from my run it's hovering around 82%.

Reply:
More precise numbers - with current develop, we have 82.6% and with this branch it's at 82.4%.

Seems to be a 1-line drop on bc_module's `model.py` file

Reply:
Sorry, forgot to follow up on this. That seems like a reasonable change.

Reply:
Still investigating the 1-line, since `model.py` wasn't changed but pytest claims there's 31 (?) lines in the file vs. 30 before.